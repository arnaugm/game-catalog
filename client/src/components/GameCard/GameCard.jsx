import React from 'react';

import STYLES from './GameCard.module.scss';

const GameCard = () => <div className={STYLES.GameCard}>Hello game!</div>;

GameCard.propTypes = {};

GameCard.defaultProps = {};

export default GameCard;
