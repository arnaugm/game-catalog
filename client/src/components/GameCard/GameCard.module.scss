.GameCard {
  font-size: larger;
  color: blue;
}
