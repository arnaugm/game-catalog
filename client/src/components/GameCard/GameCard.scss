$blue: #00f;
$font-size: larger;

.GameCard {
  color: $blue;
  font-size: $font-size;
}
