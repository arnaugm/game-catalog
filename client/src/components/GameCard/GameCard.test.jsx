import React from 'react';
import { shallow } from 'enzyme';

import GameCard from './GameCard';

describe('GameCard', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<GameCard />);
    expect(wrapper).toMatchSnapshot();
  })
});
