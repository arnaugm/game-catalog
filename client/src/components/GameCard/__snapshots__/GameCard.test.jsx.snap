// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GameCard should render correctly 1`] = `
<div
  className="GameCard"
>
  Hello game!
</div>
`;
